src/handler/http/request/prom/mod.rs

```diff
@@ -587,27 +586,15 @@ fn validate_metadata_params(
             }
         },
     };
-    let start = match start {
-        None => 0,
-        Some(s) => match parse_str_to_timestamp_micros(&s) {
-            Ok(t) => t,
-            Err(e) => {
-                let err = format!("parse time error: {e}");
-                log::error!("{err}");
-                return Err(err);
-            }
-        },
+    let start = if start.is_none() || start.as_ref().unwrap().is_empty() {
```

Comment:
Do you think we should guard against negative values of `start` and `end`?

Reply:
Yes, we can, but no none will give a negative value, there is no mean, and it can work, just can't get any result.